packages/rule-highest-available-document-mode/README.md

```diff
@@ -388,7 +388,7 @@ In the [`.sonarwhalrc`][sonarwhalrc] file:
 ```
 
 Also, note that this rule takes into consideration the [targeted
-browsers](../index.md#browser-configuration), and if Internet Explorer
+browsers](../sonarwhal/docs/user-guide/further-configuration/browser-context.md), and if Internet Explorer
```

Comment:
I don't think this will work on the site, so just to be consisted with the other places, let's use:  https://sonarwhal.com/docs/user-guide/further-configuration/browser-context/

(I'll make the changes before merging)

Reply:
Sounds good. I wasn't completely sure if it would work live since the link structure there differs from the organization in Git (haven't had a chance to walk-through how that all fits together yet).

Thanks for the quick reply!